backend/src/test/java/ca/bc/gov/hlth/mohums/integration/MoHUmsIntegrationTests.java

```diff
@@ -103,6 +105,19 @@ public void lookupUserAuthorized() throws Exception {
                 .expectStatus().isOk();
     }
 
+    @Test
+    public void addUser() throws Exception {
+        WebTestClient.ResponseSpec created = webTestClient
+                .post()
+                .uri("/users")
+                .contentType(MediaType.APPLICATION_JSON)
+                .bodyValue("{\"enabled\":true,\"attributes\":{},\"username\":\"bingo\",\"emailVerified\":\"\"}")
+                .header("Authorization", "Bearer " + jwt)
+                .exchange()
+                .expectStatus().isEqualTo(HttpStatus.CONFLICT);
+        // We expect a 409 (Conflict) because the user already exists.
```

Comment:
I know we're use to testing "successful" cases, e.g. where a user is created, but in the case of testing an external API a 409 is good: it shows that the Keycloak API accepted the request, evaluated the body, and determined that the user already exists.